test/interval_tests/numeric.jl

```diff
@@ -353,6 +353,12 @@ end
         @test sqrt(big(2..3)) == Interval(big"1.414213562373095048801688724209698078569671875376948073176679737990732478462102", big"1.732050807568877293527446341505872366942805253810380628055806979451933016908815")
     end
 
+    @testset "cbrt" begin
+        @test cbrt(2..3) == Interval(1.259921049894873, 1.4422495703074085)
+
+        @test cbrt(big(2..3)) == Interval(big"1.259921049894873164767210607278228350570251464701507980081975112155299676513956", big"1.442249570307408382321638310780109588391869253499350577546416194541687596830003")
```

Comment:
Maybe you should check that the `big` version is contained inside the float version.
In general we should probably add tests like that to the tests.